### PR TITLE
qt5-webengine: update to 5.15.11

### PR DESCRIPTION
**Pull request by Johnnynator**
[ci skip]
- qt5-webengine: update to 5.15.11.
- qt5-webengine: patch for pipewire 0.3

#### Testing the changes
- I tested the changes in this PR: **YES**|**briefly**|**NO**

### CODE REVIEW

File: srcpkgs/libpipewire0.2/template [hunk 1]
```diff
@@ -1,36 +0,0 @@
-# Template file for 'libpipewire0.2'
-pkgname=libpipewire0.2
```
Comment (classabbyamp) [inline srcpkgs/libpipewire0.2/template:2]: should be added to removed-packages

**Review by classabbyamp — COMMENTED** [review]
can we move this to python 3?

**Comment by Johnnynator** [comment]
> can we move this to python 3?

I suppose I could add a patch to it.

**Comment by classabbyamp** [comment]
It may not need any patching, chromium was able to move to py3 without anything iirc

**Comment by Johnnynator** [comment]
> It may not need any patching, chromium was able to move to py3 without anything iirc

qt5-webengine uses a rarther old chromium base, also the qt build script also explicitly check for py2.

**Pull request merged**